Add active community to Criteria (#240)
publiccodenet/standard: criteria/active-community.md

---
order: 15
---

> # Foster an active community

Reviewer: Whitespace needed before (and after) a heading, also after the front-matter.

## Requirements

* Software MUST be in use by multiple parties
* Communication channels MUST exist for users AND developers
* The roadmap SHOULD be influenced by the needs of multiple parties
* MAY hold regular events

## Why this is important

* Members of active communities support each other.

> * Codebases without communities are either too narrow for stewardship by the Foundation for Public code or must continue under incubation until a self-sustaining community can be built.

Reviewer 2: ```suggestion
* Codebases without communities are either too narrow for stewardship by the Foundation for Public Code or must continue under incubation until a self-sustaining community can be built.
```

## What this does not do

> * This does not shift responsibility for documentation, support, bug-fixing, or anything else to an unpaid community.

Reviewer 2: ```suggestion
* shift responsibility for documentation, support, bug-fixing, or anything else to an unpaid community
```

> ## How to test

Reviewer: The 'How to test' section should explain how to test for compliance on the 'Requirements', the current things below, whilst worthwhile metrics are not tests. Perhaps they should find a home under business management? Do we want the business management do this?

> * Count known users

Reviewer: ```suggestion
* The codebase is used in more than 1 context
```

* Count messages per week
* Count commits per week

> * Count distinct commit authors per week

Reviewer: * There are contributors from more than 1 context

* Graph bug time-to-resolution over time
* Count bugs and feature requests opened over time

## Policy makers: what you need to do

> * Policy makers should ensure that their organization is listed in the known users.

Reviewer 2: ```suggestion
* Make sure their organization is listed in the known users
* Encourage relevant policy makers in organizations already collaborating on a codebase to get more involved
```

> * Policy makers should ensure that their organization is listed in the known users.

Reviewer: Are there things which policy makers can create policies for, are there ways they can reach out to others or are there cultural things they can affect.

## Management: what you need to do

> * Ensure awareness of the community

Reviewer 2: Suggestions:

Make sure devs, designers and policy people have time to contribute to the community
Prioritise community maintenance work

> * Ensure awareness of the community

Reviewer: I don't really understand what this means.

> * Track and publish metrics

Reviewer: I think this point needs a bit more context; what are these metrics on and what is their goal?

## Developers and designers: what you need to do

> * Track and publish code metrics

Reviewer: There should be whitespace before the bulleted list.

## Further reading

> * [Top 5 open source community metrics to track] (https://opensource.com/business/15/12/top-5-open-source-community-metrics-track)

Reviewer: This markdown link is broken

```suggestion
* [Top 5 open source community metrics to track](https://opensource.com/business/15/12/top-5-open-source-community-metrics-track)
```
